numba/tests/test_np_functions.py

```diff
@@ -3172,6 +3176,66 @@ def test_1D_weights_axis(data, axis, weights):
         #test with axis argument
         test_1D_weights_axis(data, axis=1, weights=w)
 
+    def test_allclose(self):
+
+        pyfunc = np_allclose
+        cfunc = jit(nopython=True)(pyfunc)
+
+        a = np.asarray([1e10, 1e-7])
+        b = np.asarray([1.00001e10, 1e-8])
+        assert not cfunc(a, b)
```

Comment:
I though that since, output of `cfunc` and `pyfunc` is a `bool` object so using `assert` should be fine. Though I am fine with `self.assertEqual` as well. Will do the replacements.